### PR TITLE
ticket: #152, RSS Feed, fix CORS

### PR DESCRIPTION
**Pull request by cerberusmaster**

### CODE REVIEW

**Comment by vincentdavis** [comment]
@pprolancer
@karthicksakkaravarti reviewed the previous pull for this. Does it look ok? Can we merge to production?

File: wrh_organization/apps/cycling_org/rest_api/views.py [hunk 1]
```diff
@@ -161,6 +161,23 @@ def create(self, request, *args, **kwargs):
...
+    def get(self, request, *args, **kwargs):
+        headers = {
+            "Accept": "*/*", 
+            "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36",
+        }
+
+        content = requests.post("http://www.bicyclecolorado.org/feed/", headers=headers)
+        return HttpResponse(content.text)
+
+    def create(self, request, *args, **kwargs):
```
[the remark this answers is not in the document]
Comment (cerberusmaster) [reply wrh_organization/apps/cycling_org/rest_api/views.py:176]: Yes, we can replace the URL "http://www.bicyclecolorado.org/feed/" with request.GET["url"]

File: wrh_organization/FRONTEND/wrh_organization_ui/src/views/public/PublicOrgProfile.vue [hunk 2]
```diff
@@ -180,6 +180,12 @@
         </v-card>
       </v-col>
 
+      <v-col cols="6" v-if="organization.rss">
+        <v-card class="d-flex px-4 pt-4">
+          <VueRssParser :feedUrl="`/api/v1/cycling_org/rss?url=${organization.rss_url}`"/>
```
[the remark this answers is not in the document]
Comment (cerberusmaster) [reply wrh_organization/FRONTEND/wrh_organization_ui/src/views/public/PublicOrgProfile.vue:185]: This was my mistake - we can replace the URL "http://www.bicyclecolorado.org/feed/" with request.GET["url"]

File: wrh_organization/apps/cycling_org/rest_api/views.py [hunk 3]
```diff
@@ -161,6 +161,23 @@ def create(self, request, *args, **kwargs):
...
+class RSSFeedView(viewsets.ViewSet):
+    permission_classes = (permissions.AllowAny,)
+
+    def get(self, request, *args, **kwargs):
+        headers = {
+            "Accept": "*/*", 
+            "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36",
+        }
+
+        content = requests.post("http://www.bicyclecolorado.org/feed/", headers=headers)
```
[the remark this answers is not in the document]
Comment (cerberusmaster) [reply wrh_organization/apps/cycling_org/rest_api/views.py:173]: Yes

**Pull request closed without merging**